src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -396,22 +400,22 @@ namespace AirflowNetworkBalanceManager {
                 auto const &thisObjectName = UtilityRoutines::MakeUPPERCase(instance.key());
                 state.dataInputProcessing->inputProcessor->markObjectAsUsed(CurrentModuleObject, instance.key()); // Temporary workaround
 
-                Real64 coeff{fields.at("air_mass_flow_coefficient_at_reference_conditions")}; // Required field
-                Real64 expnt{0.65};
-                if (fields.find("air_mass_flow_exponent") != fields.end()) { // not required field, has default value
-                    expnt = fields.at("air_mass_flow_exponent");
-                }
+                Real64 coeff = state.dataInputProcessing->inputProcessor->getRealFieldValue(
+                    state, CurrentModuleObject, fields, objectSchemaProps, "air_mass_flow_coefficient_at_reference_conditions");
```

Comment:
@amirroth When trying to apply the new `getFieldValue` functions in an actual getInput function, it seemed silly to have patterns like
```
Real64 coeff = 0.0;
state.dataInputProcessing->inputProcessor->getFieldValue(state, CurrentModuleObject, fields, objectSchemaProps, "air_mass_flow_coefficient_at_reference_conditions", coeff);
```
By moving to explicitly typed functions, this reduces to one line. Seems more readable.
```
Real64 coeff = state.dataInputProcessing->inputProcessor->getRealFieldValue(state, CurrentModuleObject, fields, objectSchemaProps, "air_mass_flow_coefficient_at_reference_conditions");
```